packages/omnicounter-devtools-evm/src/omnicounter/sdk.ts

```diff
@@ -1,18 +1,9 @@
 import { EndpointId } from '@layerzerolabs/lz-definitions'
 import { IncrementOutput, IncrementType, IOmniCounter } from '@layerzerolabs/omnicounter-devtools'
-import { EndpointV2Factory } from '@layerzerolabs/protocol-devtools'
 import { OApp } from '@layerzerolabs/ua-devtools-evm'
 import { type OmniAddress, makeBytes32 } from '@layerzerolabs/devtools'
-import { OmniContract } from '@layerzerolabs/devtools-evm'
 
 export class OmniCounter extends OApp implements IOmniCounter {
-    public constructor(
-        public override contract: OmniContract,
-        protected override endpointV2Factory: EndpointV2Factory
-    ) {
-        super(contract, endpointV2Factory)
-    }
```

Comment:
Got it, because we removed endpointV2Factory, we can just rely on OApp constructor.  Leaving this here for anyone else reviewing.